### PR TITLE
Migrate from Travis to GitHub actions

### PR DESCRIPTION
**Pull request by Kobzol**

### CODE REVIEW

File: .github/workflows/check.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: Check
+
+on:
+  push:
+    branches:
+      - gh-ci
```
Comment (mikicz) [inline .github/workflows/check.yml]: I presume this is temporary for now?

**Review by mikicz — APPROVED** [review]
I haven't worked with GitHub Actions, do I need to enable them somehow for the CI to run?

File: .github/workflows/check.yml (hunk 1 above)
Comment (Kobzol) [reply .github/workflows/check.yml]: Yes, it's temporary, it should be `master`.

**Comment by Kobzol** [comment]
I don't think that you need to enable them explicitly, but I think that some workflow needs to be in the default branch before the first pipeline can execute. I suppose that you can try to merge it and then see if it appears [here](https://github.com/pyvec/naucse-hooks/actions) :)

**Comment by mikicz** [comment]
Ah yes - now it works: https://github.com/pyvec/naucse-hooks/actions/runs/1684134881

**Pull request merged**